src/custommem.c

```diff
@@ -787,21 +893,28 @@ void* internal_customRealloc(void* p, size_t size, int is32bits)
     blocklist_t* l = findBlock(addr);
     if(l) {
         size_t subsize;
-        if(l->type==BTYPE_LIST) {
+        if(l->type == BTYPE_LIST) {
```

Comment:
This is nit piking and not important, but it is strange to me that `==` is forced to hve space before and after, but `<=`, as seen a few lines below, does not...